From f9e6d4ac86b53cb3b9a8fcd95194798e7ec56e3a Mon Sep 17 00:00:00 2001
From: Harri Nieminen <moikkis@gmail.com>
Date: Sat, 4 Mar 2017 13:52:22 +0200
Subject: [PATCH 1/3] Remove useless gettext version check

Fixes following error:
autopoint: *** Missing version: please specify in configure.ac through a line 'AM_GNU_GETTEXT_VERSION(x.yy.zz)' the gettext version the package is using

--- a/configure.ac
+++ b/configure.ac
@@ -16,9 +16,8 @@
 AC_FUNC_SELECT_ARGTYPES
 AC_CHECK_FUNCS([bzero memset select strdup])
 AM_GNU_GETTEXT([external]) 
-AM_GNU_GETTEXT_VERSION
-LIBS="`fltk-config --use-images --ldflags` -lasound"
-CXXFLAGS="-g `fltk-config --use-images --cxxflags`"
+LIBS="$(fltk-config --use-images --ldflags) -lasound ${LIBS}"
+CXXFLAGS="$(fltk-config --use-images --cxxflags) ${CXXFLAGS}"
 AC_CHECK_HEADERS([unistd.h])
 AC_CHECK_HEADER(alsa/asoundlib.h, true, AC_MSG_ERROR([Alsa is required]))
 AC_PATH_PROG(FLTK_CONFIG, fltk-config, no)
